liba/src/external/openbsd/s_erff.c

```diff
@@ -0,0 +1,185 @@
+/* s_erff.c -- float version of s_erf.c.
```

Comment:
Same comment as for s_logbf.c, could you please change this file with [s_erff.c.zip](https://github.com/numworks/epsilon/files/4254712/s_erff.c.zip)?